README.md

```diff
@@ -28,10 +57,14 @@ To run the tests for Kibosh, type:
 
 # Running
 
-To run Kibosh, you supply the mount point as the first argument, and the
-directory which you would like to mirror as the second argument.
+To run Kibosh, you supply a mirror as the mount point as the first argument, 
```

Comment:
how about "you supply a mirror directory..."

Reply:
Edited as suggested.